tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/completion-tags-validator.js

```diff
@@ -0,0 +1,49 @@
+import { CleanText } from "../../../helpers/text-helpers";
+import { getSubmissionTimeText } from "../dpv-helpers/time-helpers";
+
+
+export const selfAssessmentPageNewRecommendation = (section, maturity) => {
+    const time = new Date();
+    const timePlusOneMinute = new Date(time).setMinutes(time.getMinutes() + 1);
+    const lateTime = new Date(timePlusOneMinute);
+
+    const matchingRecommendation = section.recommendation.intros.find(
+        (recommendation) => recommendation.maturity == maturity
+    );
+
+    if (!matchingRecommendation)
+        throw new Error(
+            `Couldn't find a recommendation for maturity ${maturity} in ${section.name}`,
+            section
+        );
+
+    const expectedPath =
+        `/${section.name.trim()}/recommendation/${matchingRecommendation.slug.trim()}`
```

Comment:
Would it be better to use `.../dpv-helpers/slugify-chunks.js` instead of `section.name.trim()` here?

Reply:
It would be, but the chunk slugs don't match up with the section slugs (it's hyphens - see back-up/backup broadband). We should discuss, there may be a ticket needed to bring chunk slugs in line.